ui/app/routes/variables/variable.js

```diff
@@ -7,6 +7,6 @@ export default class VariablesVariableRoute extends Route.extend(
   WithModelErrorHandling
 ) {
   model(params) {
-    return this.store.findRecord('variable', params.path);
+    return this.store.findRecord('variable', decodeURIComponent(params.path));
```

Comment:
Handle the possibility of a user pasting both `my/path/to` and `my%2fpath%2fto` correctly

Reply:
`nit`:  Should we extract this functionality into a function that creates a derived state variable.

Example:

```
const sanitizedPath = decodeURIComponent(path);

async model({path}) {
  return this.store.findRecord('variable', path);
}
```